src/test/java/api/CirculationTenantAPITests.java

```diff
@@ -0,0 +1,29 @@
+package api;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+
+import org.folio.HttpStatus;
+import org.folio.circulation.support.http.client.Response;
+import org.junit.Test;
+
+import api.support.APITests;
+import api.support.fakes.FakePubSub;
+
+public class CirculationTenantAPITests extends APITests {
+
+  @Test
+  public void tenantAPIFailsWhenCantRegisterInPubSub() {
+    FakePubSub.setFailPubSubRegistration(true);
+
+    Response response = tenantAPIFixture.postTenant();
+    assertThat(response.getStatusCode(), is(HttpStatus.HTTP_INTERNAL_SERVER_ERROR.toInt()));
+  }
+
+  @Test
+  public void tenantAPISucceedsWhenCanRegisterInPubSub() {
+    Response response = tenantAPIFixture.postTenant();
+    assertThat(response.getStatusCode(), is(HttpStatus.HTTP_CREATED.toInt()));
```

Comment:
How do we know whether the registration has registered the right set of events?

Reply:
We're checking if a correct PubSub endpoint has been called with a valid body (this was added just now).

Reply:
Thanks